src/fparser/two/tests/utils/test_blockbase.py

```diff
@@ -34,12 +34,14 @@
 
 ''' File containing unit tests for the BlockBase baseclass in utils.py '''
 
+import textwrap
 from fparser.two.utils import BlockBase
+from fparser.common.graph_tools import LineAnnotator
```

Comment:
Note: I haven't included this file yet, as it isn't really what I'm asking for feedback on. Let me know if you'd like this to run your own tests locally.